src/main/java/org/jenkinsci/test/acceptance/docker/DockerContainerHolder.java

```diff
@@ -22,15 +25,35 @@
     @Inject
     Docker docker;
 
+    @Inject
+    private FailureDiagnostics diag;
+
     T container;
 
     /**
+     * Injecting a portOffset will force the binding of dockerPorts to local Ports with an offset
+     * (e.g. bind docker 22 to localhost port 40022,
+     */
+    @Inject(optional = true)
+    @Named("dockerPortOffset")
+    private int portOffset = 0;
+
+    /**
      * Lazily starts a container and returns the instance.
      */
     @Override
     public synchronized T get() {
-        if (container==null)
-            container = docker.start((Class<T>)type.getRawType());
+        if (container==null) {
+            Class<T> fixture = (Class<T>) type.getRawType();
+            File buildlog = diag.touch("docker-" + fixture.getSimpleName() + ".build.log");
```

Comment:
Right this is a FailureDiagnostics, I missed that the first time around, so should not have this issue.